src/build-tools/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "mz-build-tools"
+description = "Provides access to tools required in build scripts."
+version = "0.0.0"
+edition.workspace = true
+rust-version.workspace = true
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Do we usually delete this comment or leave it?

Reply:
Good call, I removed it